clap_complete/src/engine/custom.rs

```diff
@@ -274,7 +274,19 @@ impl ValueCompleter for PathCompleter {
             current_dir_actual = std::env::current_dir().ok();
             current_dir_actual.as_deref()
         });
-        let mut candidates = complete_path(current, current_dir, filter);
+
+        let Some(current) = current.to_str() else {
+            return vec![];
+        };
```

Comment:
We shouldn't be doing this

Reply:
Maybe I should have used ` let mut current = current.to_string_lossy().to_string();` ?